ArjunaCore/arjuna/classes/com/arjuna/ats/internal/arjuna/recovery/PeriodicRecovery.java

```diff
@@ -12,9 +12,11 @@
 import java.util.Vector;
 
 import com.arjuna.ats.arjuna.common.recoveryPropertyManager;
+import com.arjuna.ats.arjuna.coordinator.ActionManager;
```

Comment:
I think this import can be removed now?

Reply:
At times, I wish we had the style checker active in Narayana. I will fix this, thanks for spotting it